app/kuma-cp/cmd/root.go

```diff
@@ -41,6 +50,10 @@ func newRootCmd() *cobra.Command {
 	}
 	// root flags
 	cmd.PersistentFlags().StringVar(&args.logLevel, "log-level", kuma_log.InfoLevel.String(), kuma_cmd.UsageOptions("log level", kuma_log.OffLevel, kuma_log.InfoLevel, kuma_log.DebugLevel))
+	cmd.PersistentFlags().StringVar(&args.outputPath, "output-path", args.outputPath, "path of output rotating file")
```

Comment:
I see that if we set this out then we have no logs in the stdout. @nikita15p @kumahq/kuma-maintainers do you think it's expected behavior?

If we want to stick to it, we should at least print something like `Logs will be stored in path_to_log_file` before we set the logger.

Reply:
This "Logs will be stored in path_to_log_file" can be taken into account. I liked the idea.

Reply:
Ok, let's add it then!

Reply:
Added. Please review.